Rakefile

```diff
@@ -83,6 +83,7 @@ namespace :docs do
     require "rails"
     require "action_controller"
     require "view_component"
+    ViewComponent::Base.config.view_component_path = "view_component"
```

Comment:
@boardfish what do you think about this fix?

Reply:
It's isolated to this file, and this is just a build task, so I think it's alright. I do wonder if it's symptomatic of something else that's wrong with ViewComponent itself, though. It might be worth thinking about what we use `view_component_path` for a little more widely. I reckon there's probably a justifiable reason for it, but I wonder if we could infer necessary paths based on how folks use ViewComponent and leave the rest to Zeitwerk.

Reply:
Yes, that's precisely what I was thinking. I think we should revisit the purpose and uses of this setting. But that's for another PR.